packages/react-components/src/components/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import {State, FieldJson} from '@aemforms/af-core';
+import '../../dist/theme.css';
+
+const RadioButtonGroup = (fieldjson: State<FieldJson>) => {
+    const [props, handlers] = useRuleEngine(fieldjson);
+    const { id, label, required, enumNames, enum: enums, name, valid, errorMessage, description } = props;
+    const i18n = useFormIntl();
+    const options = enumNames && enumNames.length ? enumNames : enums || [];
+    const validateState = valid === false ? 'invalid' : 'valid';
+    const error = validateState === 'invalid';
+    const formattedDescription = i18n.formatMessage({ id:`${description}` });
+    const formattedLabel = i18n.formatMessage({ id: `${label?.value}`});
+   
+    const changeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+       const val = event.target.value;
+       handlers.dispatchChange(val);
+     };
+    return (
+       <div className="cmp-adaptiveform-radiobutton">
+             <label id={`${id}-label`} htmlFor={id} className="cmp-adaptiveform-radiobutton__label">{formattedLabel}</label>
+         <div className="cmp-adaptiveform-radiobutton__widget HORIZONTAL">
```

Comment:
HORIZONTAL & VERTICAL classes should be dynamic.

Reply:
Added dynamic behaviour